custom_components/easee/services.yaml

```diff
@@ -1,40 +1,45 @@
 start:
-  description: "Deprecated HA 2023.1.0\nStarts charger. Also authorizes if configured to use authorization."
```

Comment:
It was an attempt to wrap the lines. `\n` work in parts of the frontend but obviously not here.

Reply:
Yes, it would have been nice. We should perhaps try the HTML version, i.e. <br> before we give up...

Reply:
Tried, does not work.